src/core/i18n/es/translation.es.json

```diff
@@ -909,19 +914,15 @@
     },
     "contributors": {
       "search": {
-        "title": "¡Explora los usuarios y organizaciones que están contribuyendo!",
-        "subtitle": "Introduce algún término de búsqueda. Haz click sobre las tarjetas para ver los perfiles de los resultados.",
-        "placeholder": "Buscar usuarios u Organizaciones"
+        "title": "Mira quienes están contribuyendo",
+        "subtitle": "Echa un ojo a las organizaciones y usuarios que están contribuyendo a los Desafíos, Oportunidades y Hubs en la plataforma",
+        "placeholder": "Buscar..."
```

Comment:
These probably need to be updated for nl, bg and ua.

Reply:
We need to address translations structurally. These can be left out for this version, they will default to English.